docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc

```diff
@@ -92,6 +92,27 @@ To disable this behavior, use the SPI option `spi-single-use-object-infinispan-p
 The SPI behavior of `SingleUseObjectProvider` has changed that for revoked tokens only the methods `put` and `contains` must be used.
 This is enforced by default, and can be disabled using the SPI option `spi-single-use-object-infinispan-persist-revoked-tokens`.
 
+= HA Multi-Site Deployments
+
+{project_name} 26 introduces significant improvements to the recommended HA multi-site architecture, most notably:
+
+- {project_name} deployments on each site are now able to handle user requests simulaneously, Active/Active vs the previous
+Active/Passive architecture.
+- LoadBalancing is now handled by AWS Global Accelerator, meaning there's no longer issues with DNS caching on failover/switchover
+that existed with the previous Route53 architecture.
+- {project_name} utilises Infinispan RemoteCache's to communicate with Infinispan deployments more efficiently.
```

Comment:
```suggestion
- Handling of the external Infinispan has been improved to reduce memory usage and request latency.
```

Reply:
I've reworded slightly as I don't think we should start with "Handling":

```
External Infinispan communication has been improved to reduce memory usage and request latency.
```